profile/README.md

```diff
@@ -1,32 +1,43 @@
-# Arcalot
-
-<img align="right" width="200px" style="padding-left: 2em;" alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the silhouette of a castle in the background" src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png">
-
-The Arcalot community develops tools, plugins, and libraries that you can use either standalone as a library, and/or via a user interface or CLI. You can run the tools locally, remotely, or as part of a bigger system. Arcalot:
-
-* Helps you create workflows with **normalized input and output schemas**
-* Provides you with **assisted and automated root cause analysis** for the workflows you create as well as CI and other log systems
-* Provides **stable plugins for several workloads**
+# <a href="https://arcalot.io">Arcalot</a>
+<a href="https://arcalot.io"><img align="right" width="200px" style="padding-left: 2em;"
+ alt="Arcalot logo showing a shield with the Arcalot inscription on a hill with the 
+ silhouette of a castle in the background" 
+ src="https://github.com/arcalot/.github/raw/main/branding/arcalot.png"></a>
+
+The Arcalot community is dedicated to developing modular tools, plugins, and libraries
+with flexible implementations independently or as complete end-to-end solutions. We
+believe in enabling the automation and portability of complex tasks and in
+pre-validating actions to avoid costly re-runs due to late failures and incompatible
+data.
 
 <br clear="right"/>
 
-## Arcaflow
+## <a href="https://arcalot.io/arcaflow">Arcaflow</a>
 
-<img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png">
+<a href="https://arcalot.io/arcaflow"><img align="left" width="200px" style="padding-right: 2em;" alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.png"></a>
 
-Arcaflow is a workflow engine consisting of three main components:
+Arcaflow is a workflow system consisting of three main components:
 
-* Core engine
-* UI
-* Plugins (including SDKs for Go and Python to write your own plugins)
+* A core engine, statically-compiled and platform-portable, written in Go
+* Plugins, written in various languages and delivered as Linux containers or modules
```

Comment:
"Written in various languages" is almost meaningless once we get to "Linux containers". We support Python modules and we support containers providing a specific launch command, and we build Python and Go SDKs to construct the launch framework and schema support...

Reply:
I think we want to encourage people to "bring their own language" (or, at least, we want to avoid _discouraging_ people who aren't fans of either Go or Python...).

Besides, this page is mostly marketing-speak -- it's OK if it is almost meaningless...it's all about what the _reader_ will attribute to the words.  🙂

Reply:
While "bring your own language" sounds tempting, it's more like "bring your own headache" unless you're using a Python or Go framework. Yeah, to varying extents you can call out from a "base" Python/Go plugin if only via subprocesses.

But if you decide to write a plugin in Rust, or C++, or whatever, you need to fully implement the standard plugin entrypoint script with all the options, including the ARCA schema. Working in a different language is an extremely advanced topic requiring massive internal knowledge and basically cloning one of the existing SDK modules ... and then keeping sync with the versions.

That's an incredible amount of work and I'd rather not start out by suggesting "oh, just start using whatever language you want and it'll work out".

Reply:
Your points are warranted, but but for users who are going to use off-the-shelf plugins, "various languages" will suffice.

Reply:
Users interested only (or even primarily) interested in off-the-shelf plugins don't care. For someone interested in writing their first plugin, we'd definitely want to firmly nudge them at Python or Go from the very beginning.

I'd much prefer "Plugins, delivered as Linux containers or Python modules" for a first encounter.

Reply:
The "various languages" term here is written to be both technically correct and forward-compatible. There is in fact no restriction to the language that a plugin could be written in today, and in the future we may decide to invest in SDKs for other languages. If I say "Go and Python" here then I create a docs maintenance debt for myself later. If I don't say anything and just call it containers, I'm afraid it's too ambiguous and hand-wavy.